Vagrantfile

```
@@ -5,7 +5,7 @@
 VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  os = "bento/ubuntu-18.04"
+  os = "generic/ubuntu1804"
```

```suggestion
  os = "generic/ubuntu2004"
```

I tested this out with vbox, the setup it's primarily meant for, and it works as expected. I think it should update to Ubuntu 20.04, and I don't see an issue with it being merged in. Extends functionality to other hypervisors. Supported hypervisor targets:

- libvirt: Hosted by Vagrant Cloud (1.4 GB)
- parallels: Hosted by Vagrant Cloud (1.36 GB)
- vmware_desktop: Hosted by Vagrant Cloud (1.37 GB)
- hyperv: Hosted by Vagrant Cloud (1.36 GB)
- virtualbox: Hosted by Vagrant Cloud (1.36 GB)

So, this still seems to work with the intended setup of a Virtualbox hypervisor target based on my basic testing with Virtualbox 6.1. Of course, the person setting up for a different hypervisor would be on their own, so we may need to explicitly state that in case the Vagrantfile needs further modifications when used with the other hypervisors.